tobac/utils.py

```diff
@@ -977,6 +977,18 @@ def xarray_to_iris(func):
     -------
     wrapper : function
         Function including decorator.
+
+    Examples
+    --------
+    >>> segmentation_xarray = xarray_to_iris(segmentation)
+
+    This line creates a new function that can process xarray fields and 
+    also outputs fields in xarray format, but otherwise works just like 
+    the original function:
+
+    >>> mask_xarray, features = segmentation_xarray(
+        data_xarray, dxy, threshold
```

Comment:
this is just a tiny thing but `features` are also needed as input, right? 
 --> `segmentation_xarray(features, data_xarray, dxy, threshold )`

Reply:
Yes, added that!